vit-servicing-station-lib/src/v0/endpoints/snapshot/mod.rs

```diff
@@ -115,7 +116,7 @@ mod test {
 
         assert_eq!(
             warp::test::request()
-                .path(format!("/snapshot/{}", "tag_a").as_ref())
+                .path(format!("/snapshot/snapshot_info/{}", "tag_a").as_ref())
```

Comment:
Appreciate it's not your PR adding this, but could this just be `"/snapshot/snapshot_info/tag_a"`?

Reply:
Oh, yea, will update it, no problem 😊